addons/website_sale/models/website.py

```diff
@@ -294,6 +346,17 @@ def parse_style_config(style_config_):
         for xml_id in views_to_enable:
             ThemeUtils.enable_view(xml_id)
 
+        # For selecting light palette for footer
+        Assets.make_scss_customization(
+            '/website/static/src/scss/options/colors/user_color_palette.scss',
+            {'footer': 2},
+        )
+        # For website editor to recognize the correct footer template (reason `isApplied` method of footer plugin)
+        Assets.make_scss_customization(
+            '/website/static/src/scss/options/user_values.scss',
+            {'footer-template': 'website_sale'},
+        )
```

Comment:
- Should not you check that the website_sale footer was indeed in the list to be enabled before doing that?
- Please 80 chars max per line for comments :pray: